tests/test_funcs.py

```diff
@@ -66,6 +66,20 @@ def test_recurse(self, C, dict_class):
             C(C(1, 2), C(3, 4)), dict_factory=dict_class
         )
 
+    def test_non_atomic_types(self, C):
+        """
+        Test subclasses of atomic types
```

Comment:
This doesn't conform with our docstring standard and proves why it's important :)

What exactly is tested here?  (c.f. https://hynek.me/articles/document-your-tests/ for more context)

Reply:
The method name and docstring indeed did not match, I updated the name.

I was trying to test the `else` branch inside the recursive part of the `asdict` implementation. The branch covers non-atomic objects minus some container types. I could add some more tests (e.g. `fraction.Fraction`) if needed.